docs/release_guide.md

```diff
@@ -2,12 +2,17 @@ In order to release a new version of `demisto-sdk` to the public follow these st
 
 1) Make sure the **CHANGELOG.md** file is in order and is updated with all the changes in the current release.
 2) In demisto-sdk, create a new release branch, named after the release version in the format `X.X.X`, e.g. `1.0.0`
-3) Update the version of the SDK in Demisto's Content repository by updating the demisto-sdk version in the [**dev-requirements-py3.txt**](https://github.com/demisto/content/blob/master/dev-requirements-py3.txt) file. Use the release branch first.
-4) As the regular Content build runs, initiate a nightly Content build from the Content repository by running: `./Utils/trigger_content_nightly_build.sh <branch_name> <circle_token>`
-5) Look at the CircleCI build:
-  a) Check that the unit-tests container passed sucessfully.
-  b) Check that the Test Playbooks have started running successfully, and that no erorrs occurred when setting the Demisto server instance.
-  c) Cancel the build as we do not want to run all the nightly test playbooks.
+3) Update the version of the SDK in Demisto's Content repository by updating the demisto-sdk version in the [**dev-requirements-py3.txt**](https://github.com/demisto/content/blob/master/dev-requirements-py3.txt) file. Use the release branch first - replace the `demisto-sdk==version` line with this line: `git+https://github.com/demisto/demisto-sdk.git@release-branch-name.`
+4) As the regular Content build runs, initiate the following builds:  
+  a) Trigger the nightly Content build from the Content repository by running:  
+  `./Utils/trigger_content_nightly_build.sh <branch_name> <circle_token>`.  
+  b) Trigger a build that imitates the content master build from the SDK repository by running:   `./demisto_sdk/utils/trigger_against_content_master.sh <release_branch_name> <circle_token>`
```

Comment:
why do we need them both?

Reply:
To my understanding the flows are different. 
Until now only nightly was running.

Reply:
yeah but we are not changing anything in the tests, if we are already mimicking the commands in out build why should we do anything apart from it?

@yuvalbenshalom @Itay4 @yaakovi what do you think

Reply:
@ronykoz any new thoughts?

Reply:
currently, triggering nightly has no special features besides running all tests (unit tests and test playbooks).
after content files will be valid in a way we can run validate with `--validate-all` flag, I think we should use it in nightly builds.
this change should be included under the new `validate.sh` script so triggering the special build in the SDK will cover it.